src/DetailsView/components/left-nav/details-view-left-nav.scss

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+@import '../../../common/styles/colors.scss';
+@import '../../../common/styles/common.scss';
+
+:global(.reflow-ui) {
+    .left-nav {
+        :global {
+            .ms-Nav-compositeLink {
+                &.is-selected {
+                    a {
+                        background-color: $communication-tint-20 !important;
+                    }
+                }
+                &.is-expanded,
+                &.is-expanded + ul {
+                    background-color: $communication-tint-40;
+                    .index-circle {
+                        color: $neutral-0;
+                        background: $index-circle-background;
+                        border-color: $neutral-0;
+                    }
+                }
+            }
+        }
+    }
+}
+.left-nav {
+    max-height: calc(100vh - (#{$detailsViewTotalHeaderHeight}));
+    height: calc(100vh - (#{$detailsViewTotalHeaderHeight}));
+    overflow-y: auto;
+    border-right: 1px solid $neutral-8;
+    z-index: 100;
+    background-color: $neutral-0;
+    :global {
```

Comment:
This is good! We can eventually move these to their respective components but this should help a lot!